roles/gluster_hci/tasks/hci_volumes.yml

```diff
@@ -50,7 +50,34 @@
      options: "{{ gluster_features_hci_volume_options | default(omit) }}"
   run_once: true
   with_items: "{{ gluster_features_hci_volumes }}"
-  when: gluster_features_hci_master is not defined
+  when: "gluster_features_hci_master is not defined and gluster_features_hci_volumes[0].servers is not defined"
```

Comment:
this assumes that servers are always defined for the first volume?

Reply:
This task is for normal 3 node deployment and if we are adding extra filed "servers" in gluster_features_hci_volumes to create volumes to specific nodes then this will be skipped. and next volume creation task will be executed where "servers" will be defined.

Reply:
We can remove the glusterd2_volume section completely. We no longer support glusterd2.

Reply:
glusterd2_volume section is removed.

Reply:
glusterd2_volume section is removed.